diff --git a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt
index f83033b..03b5028 100644
--- a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt
+++ b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt
@@ -157,7 +157,7 @@ class RNCWebViewManagerImpl(private val newArch: Boolean = false) {
                         }
                         mVideoView = view
                         mCustomViewCallback = callback
-                        activity.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED
+                        activity.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
                         mVideoView.systemUiVisibility = FULLSCREEN_SYSTEM_UI_VISIBILITY
                         activity.window.setFlags(
                             WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS,
@@ -202,7 +202,8 @@ class RNCWebViewManagerImpl(private val newArch: Boolean = false) {
                         mCustomViewCallback.onCustomViewHidden()
                         mVideoView = null
                         mCustomViewCallback = null
-                        activity.requestedOrientation = initialRequestedOrientation
+                        activity.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
+
                         mWebView.themedReactContext.removeLifecycleEventListener(this)
                     }
                 }
